src/scripts/toDoListsHandler.js

```diff
@@ -0,0 +1,129 @@
+import {
+  postToDoList,
+  getToDoList,
+  deleteRequest,
+  postTask
+} from "./fetchHandler";
+import { toDoList } from "./renderToDoList";
+import { toDoTask } from "./renderToDoTask";
+
+export const toDoListsHandler = async () => {
+  const addNewTaskList = getAddTaskListBtn();
+  addListenerToAddTaskBtn(addNewTaskList);
+  addListenerToTaskList();
+  const apiData = await getToDoList("api/todolists");
+  renderHandler(apiData);
+};
+
+const getAddTaskListBtn = () => {
+  return document.querySelector("#js-add-new-task-list");
+};
+
+const getInputValue = () => {
+  return document.querySelector("#js-new-task-list").value;
+};
+
+const getInput = () => {
+  return document.querySelector("#js-new-task-list");
+};
+
+const getTaskListRenderTarget = () => {
+  return document.querySelector("#js-task-list-target");
+};
+
+const getTaskRenderTarget = taskListId => {
+  return document
+    .querySelector(`[data-id="${taskListId}"]`)
+    .querySelector(".js-task-render-target");
+};
+
+const deleteTaskList = e => {
+  e.target.parentNode.parentNode.remove();
+};
+const deleteTask = e => {
+  e.target.parentNode.remove();
+};
+
+const getCurrentToDoListId = e => {
+  return e.target.parentNode.parentNode.dataset.id;
+};
+
+const getCurrentTaskId = e => {
+  return e.target.parentNode.dataset.id;
+};
+
+const renderList = (apiResponse, element) => {
+  apiResponse.forEach(response => {
+    element.insertAdjacentHTML("afterbegin", toDoList(response));
+  });
+};
+const renderTask = (apiResponse, element) => {
+  apiResponse.forEach(response => {
+    element.insertAdjacentHTML("afterbegin", toDoTask(response));
+  });
+};
+
+const clearList = element => {
+  element.innerHTML = "";
+};
```

Comment:
Jak nazwa jest clearList to IMO powinno przyjmować argument list. Zazwyczaj czyścimy listy poprzez while(list.firstElementChild) list.firstElementChild.remove()